woocommerce/wc-smooth-generator
# Prep for releasing 1.0.3

See #53

This PR adds support for generating the plugin zip using Woorelease. We can't do a full release of the plugin because it's not hosted on WordPress.org.

### Testing

- Simulate a Woorelease with
```
wr simulate --product_version 1.0.3 https://github.com/woocommerce/wc-smooth-generator/tree/prep/1.0.3
```
- This should end with an `[ERROR] Received bad code from API`
- Install the zip from the message a few lines above and verify that it works
  - eg. `Skipping upload of asset /private/var/folders/97/0sl6mkb544l008s07qwn69_w0000gn/T/wc-smooth-generator/wc-smooth-generator.zip`

```
wp wc generate orders 5
wp wc generate products 5
```

- In a terminal `cd` to the temp folder above
- Run `git diff`
- The plugin version and package.json version should be bumped
- The changelog date placeholder should be changed to today's date.

Once this is merged, I'll create a separate PR to do the actual bump.
